clients/drcachesim/common/options.cpp

```diff
@@ -36,12 +36,32 @@
 #include "droption.h"
 #include "options.h"
 
+#ifdef WINDOWS
+# define IF_WINDOWS_ELSE(x, y) (x)
+#else
+# define IF_WINDOWS_ELSE(x, y) (y)
+#endif
+
 droption_t<bool> op_offline
 (DROPTION_SCOPE_ALL, "offline", false, "Store trace files for offline analysis",
  "By default, traces are processed online, sent over a pipe to a simulator.  "
  "If this option is enabled, trace data is instead written to files in -outdir "
  "for later offline analysis.  No simulator is executed.");
 
+droption_t<unsigned int> op_num_threads
+(DROPTION_SCOPE_CLIENT, "num_threads", IF_WINDOWS_ELSE(0, 1),
```

Comment:
This needs a comment: why is it different for Windows?

Also, if the pool is a perf hit (esp w/ only 1 thread) for offline (but we expect to make it a net win with tuning, right?), should it be disabled by default on all platforms to avoid slowing everyone down?

If this really depends on online vs offline then have the default set in C++ code and describe it in the docs here.